.github/workflows/eslint.yml

```diff
@@ -1,4 +1,4 @@
-name: Eslint
+name: eslint
```

Comment:
I took a moment to standardize the naming of the workflows so that they're all small caps.